models/issue_mail.go

```diff
@@ -36,9 +36,13 @@ func mailIssueCommentToParticipants(issue *Issue, doer *User, comment *Comment,
 		return fmt.Errorf("GetParticipantsByIssueID [issue_id: %d]: %v", issue.ID, err)
 	}
 
-	// In case the issue poster is not watching the repository,
+	// In case the issue poster is not watching the repository and is active,
 	// even if we have duplicated in watchers, can be safely filtered out.
-	if issue.PosterID != doer.ID {
+	poster, err := GetUserByID(issue.PosterID)
+	if err != nil {
+		return fmt.Errorf("GetUserByID [%d]: %v", issue.PosterID, err)
+	}
+	if issue.PosterID != doer.ID && poster.IsActive == true {
```

Comment:
Do not check booleans with `==` do just simple `&& poster.IsActive {`. Should there be also check for `&& !poster.ProhibitLogin`?